### PR TITLE
90% Hide content example

### PR DESCRIPTION
**Pull request by danielmatthew**
I've documented our preferred technique to hide content visually, but still allow ATs to access.

### CODE REVIEW

File: topics/accessibility.md [hunk 1]
```diff
@@ -45,6 +45,24 @@ Users taking advantage of assistive technologies will benefit from the addition
 }
 {% endhighlight %}
 
+#### Hide Content
+As documented at the [A11y Project](http://a11yproject.com/posts/how-to-hide-content/), and used in Bootstrap itself, the `.sr-only` class is a useful one to include within a project.
```
Comment (pads) [inline topics/accessibility.md]: Don't need the first comma here.

File: topics/accessibility.md [hunk 2]
```diff
@@ -45,6 +45,24 @@ Users taking advantage of assistive technologies will benefit from the addition
 }
 {% endhighlight %}
 
+#### Hide Content
+As documented at the [A11y Project](http://a11yproject.com/posts/how-to-hide-content/), and used in Bootstrap itself, the `.sr-only` class is a useful one to include within a project. 
+
+> There are also real world situations where you want to hide elements (e.g., a form label), but you still want text to be read by a screen reader. The “clip pattern” will hide the content visually, yet provide the content to screen readers. Unlike positioning and negative text-indent, it works with rtl languages for localization.
```
Comment (pads) [inline topics/accessibility.md]: `e.g: a form label`

**Comment by pads** [comment]
:+1:

**Comment by danielmatthew** [comment]
Ah, cheers! I'll need to jog my memory...

**Pull request merged**